### PR TITLE
Created tellers to transfer diamonds into digital diamonds

### PR DESCRIPTION
**Pull request by paulpugh000**
Added two new commands, they need the `smp.createteller` permission. 
- `/smp teller deposit` Creates a villager with a "Deposit diamonds" text above it. Right clicking it: takes a diamond from the players inventory and adds balance equal to the value of the diamond type. `DIAMOND_BLOCK`s get deposited first, then `DIAMOND`s. If a person is shift right clicking, it will deposit all the diamonds in their hand into the bank.

- `/smp teller withdraw` Creates a villager with a "Withdraw diamonds" text above it. Right clicking it will withdraw 1 `DIAMOND_BLOCK` if they have more than or equal to 9 virtual diamonds. If they have less than 9, it will give a `DIAMOND`.
Players are unable to withdraw if they have a full inventory.

### CODE REVIEW

**Comment by NRProjects** [comment]
The special unicode character prevent it from building in IntelliJ

**Review by NRProjects — CHANGES_REQUESTED** [review]
Special unicode characters prevent it for building

File: src/main/java/plugins/nate/smp/commands/SMPCommand.java [hunk 1]
```diff
@@ -74,7 +87,57 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
...
+                    villager.setRotation(player.getLocation().getYaw(), player.getLocation().getPitch());
+                    villager.setInvulnerable(true);
+                    villager.setAI(false);
+                });
+
+                player.getWorld().spawn(player.getLocation().add(0, 2.4, 0), TextDisplay.class, (text) -> {
+                    // Storing UUID of the villager it's bound to
+                    text.getPersistentDataContainer().set(SMPUtils.PARENT_TELLER_KEY, PersistentDataType.STRING, teller.getUniqueId().toString());
+                    text.setBillboard(Billboard.CENTER);
+                    text.setText("ᴅᴇᴘᴏsɪᴛ ᴅɪᴀᴍᴏɴᴅs (Đ)");
```
Comment (NRProjects) [inline src/main/java/plugins/nate/smp/commands/SMPCommand.java:115]: Special unicode characters prevent it from building

**Review by NRProjects — CHANGES_REQUESTED** [review]
Re-reviewed the correct way

File: src/main/java/plugins/nate/smp/commands/SMPCommand.java [hunk 2]
```diff
@@ -112,7 +112,7 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                     // Storing UUID of the villager it's bound to
                     text.getPersistentDataContainer().set(SMPUtils.PARENT_TELLER_KEY, PersistentDataType.STRING, teller.getUniqueId().toString());
                     text.setBillboard(Billboard.CENTER);
-                    text.setText("ᴅᴇᴘᴏsɪᴛ ᴅɪᴀᴍᴏɴᴅs (Đ)");
+                    text.setText("ᴅᴇᴘᴏsɪᴛ ᴅɪᴀᴍᴏɴᴅs ("+ '\u00D0' + ")");
```
Comment (NRProjects) [inline src/main/java/plugins/nate/smp/commands/SMPCommand.java:115]: All characters are preventing it from building

File: src/main/java/plugins/nate/smp/commands/SMPCommand.java [hunk 3]
```diff
@@ -130,7 +130,7 @@ public boolean onCommand(@NotNull CommandSender sender, @NotNull Command command
                     // Storing UUID of the villager it's bound to                    
                     text.getPersistentDataContainer().set(SMPUtils.PARENT_TELLER_KEY, PersistentDataType.STRING, teller.getUniqueId().toString());
                     text.setBillboard(Billboard.CENTER);
-                    text.setText("ᴡɪᴛʜᴅʀᴀᴡ ᴅɪᴀᴍᴏɴᴅs (Đ)");
+                    text.setText("ᴡɪᴛʜᴅʀᴀᴡ ᴅɪᴀᴍᴏɴᴅs ("+ '\u00D0' + ")");
```
Comment (NRProjects) [inline src/main/java/plugins/nate/smp/commands/SMPCommand.java:133]: All characters are preventing it from building

**Review by NRProjects — CHANGES_REQUESTED** [review]

File: src/main/java/plugins/nate/smp/commands/SMPCommand.java (hunk 3 above)
Comment (paulpugh000) [reply src/main/java/plugins/nate/smp/commands/SMPCommand.java:133]: Can you post logs of what errors you are getting? Building with Gradle works perfectly fine for me.

**Review by NRProjects — APPROVED** [review]

**Pull request merged**